src/java/org/apache/cassandra/service/paxos/uncommitted/PaxosStateTracker.java

```diff
@@ -64,23 +67,26 @@
     private static final Logger logger = LoggerFactory.getLogger(PaxosStateTracker.class);
 
     // when starting with no data, skip rebuilding uncommitted data from the paxos table
-    static final String SKIP_REBUILD_PROP = "cassandra.skip_paxos_state_rebuild";
-    static final String FORCE_REBUILD_PROP = "cassandra.force_paxos_state_rebuild";
-    static final String TRUNCATE_BALLOT_METADATA_PROP = "cassandra.truncate_ballot_metadata";
+    @Deprecated
```

Comment:
why are these deprecated? we just changed where we are getting their values (from property key).

Reply:
Fixed.